cms/djangoapps/contentstore/views/course.py

```diff
@@ -11,21 +11,13 @@
 from django.contrib.auth.decorators import login_required
 from django.core.exceptions import PermissionDenied
 from django.core.urlresolvers import reverse
-<<<<<<< HEAD
-from django.http import HttpResponseBadRequest, HttpResponseNotFound, HttpResponse, Http404
-from smtplib import SMTPException
-from util.json_request import JsonResponse, JsonResponseBadRequest
-from util.date_utils import get_default_time_display
-from edxmako.shortcuts import render_to_response
-from edxmako.shortcuts import render_to_string
-=======
```

Comment:
It looks like some of this was refactored.
All of these, except `SMTPException` (see below) were duplicated elsewhere.